module Auth
  # thrown in the event that authorization has failed
  class AuthorizationException < RuntimeError

  end
end